statistics/statistics.rb

```diff
@@ -0,0 +1,36 @@
+module Statistics
+  def top_readers(orders, quantity = 1)
+    get_statistics(orders, 'reader', quantity)
+  end
+
+  def top_books(orders, quantity = 1)
+    get_statistics(orders, 'book', quantity)
+  end
+
+  def unique_readers_number(orders, quantity = 3)
+    top_books = get_statistics(orders, 'book', quantity).map!(&:title)
```

Comment:
why do you use `map!`, but not `map`

Reply:
why do you use attribute of book for mapping?
you should operate with objects.

Each object has its own id given from ruby. So it's not necessary to use attributes